v2/concordium/types.proto

```diff
@@ -1352,6 +1352,9 @@ message AccountTransactionEffects {
     repeated DelegationEvent events = 1;
   }
 
+  message AccountSuspended {}
```

Comment:
Not sure if there should be some meta info in the transaction event.

Reply:
I would suggest something like `BakerSuspended`, and have the `baker_id` as with other `BakerEvent`s.